core/application/impl/app_configuration_impl.cpp

```diff
@@ -775,6 +777,7 @@ namespace kagome::application {
         ("keystore", po::value<std::string>(), "required, node keystore")
         ("tmp", "Use temporary storage path")
         ("database", po::value<std::string>()->default_value("rocksdb"), "Database backend to use [rocksdb]")
+        ("db-cache", po::value<uint32_t>()->default_value(def_db_cache_size), "Limit the memory the database cache can use <MiB>")
```

Comment:
Please mention that default is 1024 mb

Reply:
Discussed offline - no actions required. The help message already contains that info